Source/WTF/wtf/PlatformEnable.h

```diff
@@ -876,7 +876,7 @@
 #endif
 
 /* CSS Selector JIT Compiler */
-#if !defined(ENABLE_CSS_SELECTOR_JIT) && ((CPU(X86_64) || CPU(ARM64)) && ENABLE(JIT) && (OS(DARWIN) || OS(WINDOWS) || PLATFORM(GTK) || PLATFORM(WPE)))
+#if !defined(ENABLE_CSS_SELECTOR_JIT) && ((CPU(X86_64) || CPU(ARM64)) && ENABLE(JIT) && (OS(DARWIN) || PLATFORM(GTK) || PLATFORM(WPE)))
```

Comment:
Feel like the OS and PLATFORM checks can go and just rely on whether or not JIT is enabled. @Constellation thoughts?

Reply:
Sounds good to me. We must enable it only for ARM64 and x64 and that's the most critical condition.